Ja translation

hkurosawa opened the pull request.

hkurosawa committed:
add japanese translation
- of_philosophy
- cplusplus_basics

hkurosawa committed:
日本語の調整

hkurosawa committed:
表現の調整

hkurosawa committed:
日本語の微調整

hkurosawa committed:
翻訳漏れの追加

hkurosawa committed:
modifying tags & minor translation fix

edap commented:
@hiroMTB how does it sounds to you?

hiroMTB commented:
Great work. 
I think it's mergable already.
It sounds bit "direct translation" but good point to start. We can update whenever we want.

hiroMTB commented:
I believe @tado is the right person to ask.

hkurosawa committed:
訳文の見直し

kkshmz commented:
I think we can make it more reader friendly like 

これは全世界的なプロジェクトであり、合衆国で目を覚ました時に、アジアやヨーロッパのプログラマからのプルリクエストやイシューで一杯の電子メールの受信箱を目にすることは日常茶飯事です。

to something like 
これは世界的なプロジェクトであり、アメリカ合衆国住まいのプログラマーが目を覚ましたときに、昨晩中にアジアやヨーロッパから来た様々のプルリクエストやイシューを目にすることは日常茶飯事です。

Anyways, I can help as well.

hkurosawa committed:
Update chapter.ja.md

changed translation according to review 
https://github.com/openframeworks/ofBook/pull/301#issuecomment-536855383

hkurosawa commented:
I've updated the sentence as the suggestion above. Thanks.
